packages/agoric-cli/lib/start.js

```diff
@@ -463,9 +463,14 @@ export default async function startMain(progname, rawArgs, powers, opts) {
         // eslint-disable-next-line no-await-in-loop
         exitStatus = await capret[0];
         if (!exitStatus) {
-          const ret = JSON.parse(capret[1]);
-          if (ret.code !== 0) {
-            exitStatus = 2;
+          try {
+            const ret = JSON.parse(capret[1]);
+            if (ret.code !== 0) {
+              exitStatus = 2;
+            }
```

Comment:
Solid.

It doesn’t matter during daemon startup, but the surrounding function will run deoptimized in V8. In prior review circles I’ve been in, a performance best practice was to isolate `try/catch` to trivial utility functions.